### PR TITLE
#1003 error when apply 3 d to empty canvas

### PR DESCRIPTION
**Pull request by Alex-Work-Account**

### CODE REVIEW

**Review by AnaMoskvina — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/state/shared.js [hunk 1]
```diff
@@ -75,6 +75,10 @@ export function load(struct, options) {
     return parseStruct(struct, server, options)
       .then(
         struct => {
+          if (!struct) {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/state/shared.js:78]: It looks like workaround. I guess 'then' expression  should be only executed if struct is not empty. Could you  find the reason of this case?

**Review by AndreiMazol — COMMENTED** [review]
need to  discuss

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
see comments

**Review by karen-sarkisyan — APPROVED** [review]

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
you  should not return hardcoded cml file.

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**